src/harness/simple_crl_server.py

```diff
@@ -182,17 +177,8 @@ def revokeCertificate():
   # Retrives the all certificates from certs directory.
   cert_name = getCertificateNameToBlacklist()
   logging.info('Certificate selected to blacklist is:%s', cert_name)
-  print "select the certificate type:"
-  print "\n".join("[%s] %s" % (cert_id, cert_type)
-                  for (cert_id, cert_type) in sorted(CERT_TYPE.iteritems()))
-  cert_type = raw_input(CLI_PROMPT_TEXT)
-  if cert_type not in CERT_TYPE:
-    raise Exception("RunTimeError:Type of certificate to be blacklisted is incorrect."
-                    "Please select one of the values:%s" % CERT_TYPE.values())
-
-  revoke_cert_command = "cd {0} && ./revoke_and_generate_crl.sh " \
-                        "{1} {2}".format(getCertsDirectoryAbsolutePath(),
-                                         CERT_TYPE[cert_type],
+  revoke_cert_command = "cd {0} && bash ./revoke_and_generate_crl.sh " \
```

Comment:
What's the motivation behind the explicit `bash` here?  The script already starts with `#!/bin/bash`.

Reply:
Yes   revert it back to previous state